src/js/entities-service/entities/actions.js

```diff
@@ -250,6 +249,11 @@ const _Model = BaseModel.extend({
 
     return !!size(programAction.get('allowed_uploads'));
   },
+  addAttachment(resource) {
```

Comment:
Are the names of these clear? Could cause confusion in the future (for me at least).

Maybe they should be `addAttachmentRelationship()`/`removeAttachmentRelationshop()` or `addFileRelationship()`/`removeFileRelationship()` 🤔